code/modules/surgery/organs/lavaland.dm

```diff
@@ -0,0 +1,367 @@
+			H.a_intent_change(INTENT_DISARM)
+		next_command = world.time + cooldown_meme
+
+	//GRAB INTENT
+	else if(text_in_list(message, list("grab")))
+		for(var/mob/living/carbon/human/H in listeners)
+			H.a_intent_change(INTENT_GRAB)
+		next_command = world.time + cooldown_meme
+
+	//HARM INTENT
+	else if(text_in_list(message, list("harm","fight")))
+		for(var/mob/living/carbon/human/H in listeners)
+			H.a_intent_change(INTENT_HARM)
+		next_command = world.time + cooldown_meme
+
+	//THROW/CATCH
+	else if(text_in_list(message, list("throw","catch")))
+		for(var/mob/living/carbon/C in listeners)
+			C.throw_mode_on()
+		next_command = world.time + cooldown_meme
+
+	//FLIP
+	else if(text_in_list(message, list("flip","rotate","revolve","roll","somersault")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("flip")
+		next_command = world.time + cooldown_meme
+
+	//REST
+	else if(text_in_list(message, list("rest")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			if(!L.resting)
+				L.lay_down()
+		next_command = world.time + cooldown_meme
+
+	//GET UP
+	else if(text_in_list(message, list("get up")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			if(L.resting)
+				L.lay_down() //aka get up
+			L.SetStunned(0)
+			L.SetWeakened(0)
+			L.SetParalysis(0) //i said get up i don't care if you're being tazed
+		next_command = world.time + cooldown_damage
+
+	//SIT
+	else if(text_in_list(message, list("sit")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			for(var/obj/structure/chair/chair in get_turf(L))
+				chair.buckle_mob(L)
+				break
+		next_command = world.time + cooldown_meme
+
+	//STAND UP
+	else if(text_in_list(message, list("stand")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			if(L.buckled && istype(L.buckled, /obj/structure/chair))
+				L.buckled.unbuckle_mob(L)
+		next_command = world.time + cooldown_meme
+
+	//DANCE
+	else if(text_in_list(message, list("dance")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("dance")
+		next_command = world.time + cooldown_meme
+
+	//JUMP
+	else if(text_in_list(message, list("jump")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.say("HOW HIGH?!!")
+			L.emote("jump")
+		next_command = world.time + cooldown_meme
+
+	//SALUTE
+	else if(text_in_list(message, list("salute")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("salute")
+		next_command = world.time + cooldown_meme
+
+	//PLAY DEAD
+	else if(text_in_list(message, list("play dead")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("deathgasp")
+		next_command = world.time + cooldown_meme
+
+	//PLEASE CLAP
+	else if(text_in_list(message, list("clap","applaud")))
+		for(var/V in listeners)
+			var/mob/living/L = V
+			L.emote("clap")
+		next_command = world.time + cooldown_meme
+
+	//HONK
+	else if(text_in_list(message, list("honk")))
+		playsound(get_turf(owner), "sound/items/bikehorn.ogg", 300, 1)
```

Comment:
this honk should be delayed until after the loud general sound effect is finished playing.

HONK. *beat* *honk!*